File: PHP/crud.php
<?php
//CREAR CONEXIÓN A LA BASE DE DATOS
$conn = mysqli_connect("localhost", "root", "", "personas");

//VERIFICAR LA CONEXIÓN
if (!$conn) {
    die("Conexión fallida: " . mysqli_connect_error());
}

//CREAR LA TABLA SI NO EXISTE
$sql = "CREATE TABLE IF NOT EXISTS persona (
    ID INT UNSIGNED AUTO_INCREMENT PRIMARY KEY,
    CEDULA VARCHAR(10) NOT NULL,
    NOMBRE VARCHAR(10) NOT NULL,
    APELLIDO VARCHAR(10) NOT NULL
)";

if (mysqli_query($conn, $sql)) {
    echo "Tabla creada exitosamente";
} else {
    echo "Error al crear la tabla: " . mysqli_error($conn);
}

//INSERTAR DATOS
$sql = "INSERT INTO persona (ID, CEDULA, NOMBRE, APELLIDO) VALUES (2, '1804372432','JUAN', 'CASTRO')";

if (mysqli_query($conn, $sql)) {
    echo "Registro agregado exitosamente";
} else {
    echo "Error al agregar el registro: " . mysqli_error($conn);
}

//LEER DATOS
$sql = "SELECT * FROM persona";
$result = mysqli_query($conn, $sql);

if (mysqli_num_rows($result) > 0) {
    while ($row = mysqli_fetch_assoc($result)) {
        echo "ID: " . $row["ID"] . " - CEDULA: " . $row["CEDULA"] . " - NOMBRE: " . $row["NOMBRE"] . " - APELLIDO: " . $row["APELLIDO"] . "<br>";
    }
} else {
    echo "No se encontraron registros";
}

//ACTUALIZAR DATOS
$sql = "UPDATE persona SET NOMBRE = 'CARLOS' WHERE NOMBRE = 'JUAN'";

if (mysqli_query($conn, $sql)) {
    echo "Registro actualizado exitosamente";
} else {
    echo "Error al actualizar el registro: " . mysqli_error($conn);
}

//ELIMINAR DATOS
$sql = "DELETE FROM persona WHERE NOMBRE = 'CARLOS'";

if (mysqli_query($conn, $sql)) {
    echo "Registro eliminado exitosamente";
} else {
    echo "Error al eliminar el registro: " . mysqli_error($conn);
}

//CERRAR LA CONEXIÓN
mysqli_close($conn);

?>

File: Python/crud.py
import mysql.connector

#CREAR CONEXIÓN A LA BASE DE DATOS
conn = mysql.connector.connect(
    host="localhost",
    user="root",
    password="",
    database="personas"
)

#CREAR LA TABLA SI NO EXISTE
cursor = conn.cursor()
cursor.execute("CREATE TABLE IF NOT EXISTS persona (ID INT AUTO_INCREMENT PRIMARY KEY, CEDULA VARCHAR(10), NOMBRE VARCHAR(50), APELLIDO VARCHAR(50))")

#INSERTAR DATOS
sql = "INSERT INTO persona (ID, CEDULA, NOMBRE, APELLIDO) VALUES (%s, %s, %s, %s)"
values = (2, "1804372432", "JUAN", "CASTRO")
cursor.execute(sql, values)
conn.commit()

#ACTUALIZAR DATOS
sql = "UPDATE persona SET NOMBRE = %s WHERE NOMBRE = %s"
values = ("CARLOS", "JUAN")
cursor.execute(sql, values)
conn.commit()

#ELIMINAR DATOS
sql = "DELETE FROM persona WHERE NOMBRE = %s"
values = ("CARLOS",)
cursor.execute(sql, values)
conn.commit()

#LEER DATOS
cursor.execute("SELECT * FROM persona")
rows = cursor.fetchall()
for row in rows:
    print(row)

#CERRAR LA CONEXIÓN
cursor.close()
conn.close()